apps/main/src/components/PageCreatePool/TokensInPool/index.tsx

```diff
@@ -181,13 +231,53 @@ const TokensInPool = ({ curve, chainId, haveSigner }: Props) => {
               ...updatedFormValues[TOKEN_B],
               ngAssetType: 0,
               address: value,
+              symbol: findSymbol(value),
               basePool: true,
             },
             [TOKEN_A]: {
               ...updatedFormValues[TOKEN_A],
               address: '',
             },
           }
+          // value = basepool while token in basepool is selected
+        } else if (
+          swapType === STABLESWAP &&
```

Comment:
this code looks almost identical as the other branches. it's hard to see the differences :thinking: